zun/container/oci.py

```diff
@@ -30,6 +34,14 @@ def merge_oci_runtime_config(parent, *cfgs):
         cfg_devices = cfg.get('linux', {}).get('devices')
         if cfg_devices is not None:
             devices.extend(cfg_devices)
+        cfg_capabilities = cfg.get('capabilities', {})
+        if cfg_capabilities is not None:
```

Comment:
How can this be `None`? Maybe better to make the above line `cfg.get('capabilities') or {}` instead, so it handles if it's explicitly set to None.